docs/sdks/server-sdks/python.md

```diff
@@ -8,39 +8,90 @@ Eppo's Python SDK can be used for both feature flagging and experiment assignmen
 - [GitHub repository](https://github.com/Eppo-exp/python-sdk)
 - [PyPI package](https://pypi.org/project/eppo-server-sdk/)
 
-## 1. Install the SDK
+## 1. Getting started
 
-Install the SDK with PIP:
+### A. Install the SDK
+
+First, install the SDK with PIP:
 
 ```bash
 pip install eppo-server-sdk
 ```
 
-## 2. Initialize the SDK
+### B. Initialize the SDK
 
-Initialize the SDK with a SDK key, which can be generated in the Eppo interface. Initialization the SDK when your application starts up to generate a singleton client instance, once per application lifecycle:
+To initialize the SDK, you will need a SDK key. You can generate one [in the flag interface](https://eppo.cloud/feature-flags/keys).
 
 ```python
 import eppo_client
 from eppo_client.config import Config
 
-client_config = Config(api_key="<YOUR_API_KEY>", assignment_logger=AssignmentLogger())
+client_config = Config(api_key="<YOUR_API_KEY>")
 eppo_client.init(client_config)
+client = eppo_client.get_instance()
+…
 ```
 
-After initialization, the SDK begins polling Eppo’s API at regular intervals to retrieve the most recent experiment configurations such as variation values and traffic allocation. The SDK stores these configurations in memory so that assignments are effectively instant. If you are using the SDK for experiment assignments, make sure to pass in an assignment logging callback (see [section](#define-an-assignment-logger-experiment-assignment-only) below).
+This generates a singleton client instance, that can be reused throughout the application lifecycle
 
-:::info
+### C. Assign variations
 
-By default the Eppo client initialization is asynchronous to ensure no critical code paths are blocked. For more information on handling non-blocking initialization, see our [documentation here](/feature-flagging/common-issues#3-not-handling-non-blocking-initialization).
+This instance can then be called to assigning users to flags or experiments using the `get_string_assignment` function:
+
+```python
+…
+variation = client.get_string_assignment("<SUBJECT-KEY>", "<FLAG-OR-EXPERIMENT-KEY>")
+if variation = "fast_checkout":
+    …
+elif:
+    …
+```
+
+* `<SUBJECT-KEY>` is the value if the, typically `user_id`
+* `<FLAG-OR-EXPERIMENT-KEY>` is the key that you chose when creating a flag; you can find it on the [flag page](https://eppo.cloud/feature-flags).
+
+That’s it: You can already start changing the feature flag on the page and see how it controls your code!
+
+## 2. Running the SDK
```

Comment:
HUGE fan of this new layout, it's such a massive improvement. We want to show "hey this is really easy", then "you can understand how it works and trust it", and then round out with "here are all of the advanced use cases you might be considering"

We should do the same for all of the SDK docs (can start with the most common ones)